src/xeus_sql_interpreter.cpp

```diff
@@ -347,13 +356,879 @@ namespace xeus_sql
         }
         return ok();
     }
-
-    nl::json interpreter::complete_request_impl(const std::string& /*code*/,
-                                                int /*cursor_pos*/)
+    nl::json interpreter::complete_request_impl(const std::string& raw_code,
+                                                int cursor_pos)
     {
-        nl::json jresult;
-        jresult["status"] = "ok";
-        return jresult;
+        static const std::array<std::string, 826> keywords =  
```

Comment:
Nitpicking: I think this array shoud be defined in a dedicated function at the end of the file, to improve readability (and avoid scrolling when you want to check what is defined below).

Reply:
we can return a const reference from the function since static const right?

Reply:
https://github.com/DerThorsten/xeus-sql/blob/32eb644c0b4303adb29ccce1e876d38d3f1ace4a/src/xeus_sql_interpreter.cpp#L472 for some reason the diff in this PR is not properly updated

Reply:
> we can return a const reference from the function since static const right?

Yes

Reply:
Thank you! :)
I'll merge this :rainbow: